packages/guilded-api-typings/lib/v1/ws/Events.ts

```diff
@@ -75,6 +77,8 @@ export interface WSTeamMemberRemovedPayload extends SkeletonWSPayload {
     d: {
         serverId: string;
         userId: string;
+        isKick: boolean;
```

Comment:
Being picky with verbiage but `isKicked` and `isBanned` sound better to me.

Reply:
Should we change message#deleted to message#isDeleted then?

Reply:
It might be good to keep consistent with that, yes.

Reply:
Wait haha, I misunderstood. This isn't something we can change, this is coming straight from the API so that's what the API calls it.